src/mixtures/mixturemodel.jl

```diff
@@ -347,13 +347,13 @@ function _mixlogpdf!(r::AbstractArray, d::AbstractMixtureModel, x)
     @inbounds for i = 1:K
         if p[i] > 0.0
             lp_i = view(Lp, :, i)
-            for j = 1:n
-                r[j] += exp(lp_i[j] - m[j])
+            for (j, lp_ij) in zip(axes(r, 1), lp_i)
```

Comment:
This seems wrong as well? Maybe rather
```suggestion
            for (j, lp_ij) in zip(eachindex(r), lp_i)
```